apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManagerIT.java

```diff
@@ -32,6 +34,12 @@
   private static final String SAMPLE_BENE = "567834";
   private static final String INVALID_BENE = "1";
 
+  @BeforeClass
+  public static void beforeAll() {
```

Comment:
perhaps this method should be called cleanupDatabaseBeforeTestSuite to be more consistent with the naming of the After method. same comment applies to the other classes. Also missing a javadoc on this method (with a javadoc comment on the method I think you can remove the one below on the code since it will be redundant).